src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -365,20 +394,34 @@ def run(self, batch_inputs):
             ),
         ):
             try:
+
+                base_args = (self._inputs_queue, self._line_timeout_sec, result_list)
```

Comment:
We'd better explain what is the meaning of such arguments.

Reply:
fixed